cognite/client/data_classes/datapoints.py

```diff
@@ -171,11 +172,12 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
             dumped = {utils._auxiliary.to_camel_case(key): value for key, value in dumped.items()}
         return {key: value for key, value in dumped.items() if value is not None}
 
-    def to_pandas(self, column_names="externalId") -> "pandas.DataFrame":
+    def to_pandas(self, column_names: str = "externalId", include_aggregate_name: bool = True) -> "pandas.DataFrame":
```

Comment:
`include_aggregate_name` is not used anywhere so it won't have any effect.

Reply:
Should have a unittest for this.

Reply:
nice catch. fixed and added tests all around